### ISSUE TITLE
PDF export of recipe cards

### ISSUE THREAD

**Issue by lachholden** [lachholden/obsidian-recipe-view#6]
It's nice to share recipes with friends, and it seems possible to do better than just exporting the original markdown as PDF – this results in weirdness with thumbnails, properties missing, and wasted space where there could be a two-column view.

**Comment by Eneswar** [comment]
Didnt even think about this idea, awesome.

**Comment by niki1423** [comment]
I was just about to put forward this idea when i saw this planned enhancement. This feature would make this plugin absolutely perfect! Thanks for all your work!

**Comment by Nestor-Ferreira** [comment]
Sí! estaba a punto de solicitar que se añada una función de exportación a PDF, debido a que obsidian bloquea los comandos de exportar (seguramente porque el plugin no utiliza la Reading View)
No voy a llevar mi PC cuando me junte con amigos, o incluso si voy a mi cocina!, ni tampoco tengo obsidian en el celular, por lo que me sirve imprimir la receta en papel. Pero ahora la única opción es tomar screenshot y usar algún gestor de archivos para imprimir, o pasar la screenshot a mi whatsapp
